opentelemetry-sdk/Cargo.toml

```diff
@@ -57,10 +57,6 @@ rt-async-std = ["async-std"]
 name = "context"
 harness = false
 
-[[bench]]
-name = "key_value_map"
```

Comment:
Goodbye to our trusted benchmark code. Thanks for the speedy (nanoseconds!) memories. Rest in peace!